frontend/app/src/main/java/dk/scheduling/schedulingfrontend/pages/CreateTaskPage.kt

```diff
@@ -0,0 +1,221 @@
+)
+
+package dk.scheduling.schedulingfrontend.pages
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Button
+import androidx.compose.material3.Card
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.material3.TimePickerState
+import androidx.compose.material3.rememberTimePickerState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import dk.scheduling.schedulingfrontend.components.DateRange
+import dk.scheduling.schedulingfrontend.components.FilledButton
+import dk.scheduling.schedulingfrontend.components.StandardDateRangePicker
+import dk.scheduling.schedulingfrontend.components.StandardTimePickerDialog
+import dk.scheduling.schedulingfrontend.components.Title
+import dk.scheduling.schedulingfrontend.sharedcomponents.StandardDropDownMenu
+import dk.scheduling.schedulingfrontend.ui.theme.SchedulingFrontendTheme
+
+@Composable
+fun CreateTaskPage(
+    modifier: Modifier = Modifier,
+    handleSubmission: () -> Unit,
+    handleCancellation: () -> Unit,
+) {
+    Title(titleText = "Create Task", topMargin = 0.dp)
+
+    Column(
+        modifier =
+            modifier
+                .fillMaxSize()
+                .padding(10.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.SpaceBetween,
+    ) {
+        // Dropdown box of devices
+        Spacer(modifier = Modifier.height(100.dp))
+
+        val options = listOf("Washer", "Dryer", "Toaster")
+        var selectedItem by remember { mutableStateOf(options[0]) }
+        StandardDropDownMenu(
+            modifier = Modifier,
+            options = options,
+            label = "Devices",
+            selectedItem = selectedItem,
+            onSelect = { selectedItem = it },
+        )
+
+        // Duration input field
+        Spacer(modifier = Modifier.height(20.dp))
```

Comment:
Inconsistent spacer heights.
Can we use something other than spacers for everything?
Padding works better.

Reply:
Padding is also applied on the sides, which we dont want, also due to the way the title is created we need a spacer in the top anyway

Reply:
You can specify which direction to pad with e.g. `.padding(bottom = 20.dp)`

Reply:
@mads256h no more spacers :)